Events/Ch11.event

```diff
@@ -0,0 +1,720 @@
+UNIT EnemyID Ranger   LeaderID Level(        1, Enemy, True) [10,  5] 0b 0x00 0 0x0 [SteelBow]      PursueWithoutHeedAI
+UNIT EnemyID MageKnight LeaderID Level(      1, Enemy, True) [ 9,  4] 0b 0x00 0 0x0 [Thunder, Heal] PursueWithoutHeedAI
+UNIT
+
+ALIGN 4
+Chesdrake_10_10:
+LoadReinforcements(SLuManakete_10_10)
+NoFade
+ENDA
+
+ALIGN 4
+Chesdrake_19_6:
+LoadReinforcements(SLuManakete_19_6)
+NoFade
+ENDA
+
+ALIGN 4
+ThiefRgt:
+LoadReinforcements(RgtThief)
+NoFade
+ENDA
+
+ALIGN 4
+ThiefBot:
+LoadReinforcements(BotThief)
+NoFade
+ENDA
+
+ALIGN 4
+ThiefTop:
+LoadReinforcements(TopThief)
+NoFade
+ENDA
+
+TopThief:
+UNIT C11E(Thief)   [19, 0] 0b 0x00 1 REDA16R0 [SilverSword, ChestKey] [0x06, 0x05]
+UNIT
+
+BotThief:
+UNIT C11E(Thief)   [19,25] 0b 0x00 1 REDA15R25 [SilverSword, ChestKey] [0x06, 0x05]
+UNIT
+
+RgtThief:
+UNIT C11E(Thief)   [22,12] 0b 0x00 1 REDA20R11 [SilverSword, ChestKey] [0x06, 0x05]
+UNIT
+
+ThroneMages1:
+UNIT C11E(Mage)   [ 8, 1] 0b 0x00 1 REDA8R2 [Fire] PursueWithoutHeedAI
+UNIT C11E(Monk)   [10, 1] 0b 0x00 1 REDA10R2 [Lightning] PursueWithoutHeedAI
+UNIT
+
+ThroneMages2:
+UNIT C11E(Shaman)   [ 8, 1] 0b 0x00 1 REDA8R2 [Flux] PursueWithoutHeedAI
+UNIT C11E(Monk)   [10, 1] 0b 0x00 1 REDA10R2 [Lightning] PursueWithoutHeedAI
+UNIT
+
+ThroneMages3:
+UNIT C11E(Shaman)   [ 8, 1] 0b 0x00 1 REDA8R2 [Flux] PursueWithoutHeedAI
+UNIT C11E(Mage)   [10, 1] 0b 0x00 1 REDA10R2 [Fire] PursueWithoutHeedAI
+UNIT
+
+ThroneMages4:
+UNIT C11E(Mage)   [ 8, 1] 0b 0x00 1 REDA8R2 [Fire] PursueWithoutHeedAI
+UNIT C11E(Shaman)   [10, 1] 0b 0x00 1 REDA10R2 [Flux] PursueWithoutHeedAI
+UNIT
+
+ALIGN 4
+SweetLu_Recruit:
+MUSI 
+TEXTSTART
+TEXTSHOW LuutConversation
+TEXTEND
+MUSS Comrades
+TEXTCONT
+TEXTEND
+TurnAlly(Luut)
+NoFade
+ENDA
+
+//If you like not smashing your head in, you may want to stop reading ahead.
+    ReinLeftIfInRight:
+    RANDOMNUMBER 0x8       //Sets slot C to a number from 0 to 8? smilies says 9
+    //Now get remainder mod 4 (possibilities: 0, 1, 2, 3)
+    SMOV r1 0x4
+    SDIV slotsParam(r2, rC, r1) //r2 = rC/4
+    SMUL slotsParam(r1, r2)     //r1 = r2*4
+    SSUB slotsParam(r1, rC, r1)     //r1 -= rC, r1 = remainder
+    BNE 0x24 rC 0
+        LoadReinforcements(RD0)    //Remainder was 0
+        GOTO 0x27
+    LABEL 0x24
+    SMOV r2 1
+    BNE 0x25 rC r2
+        LoadReinforcements(RD1) //r = 1
+        GOTO 0x27
+    LABEL 0x25
+    SMOV r2 2
+    BNE 0x26 rC r2
+        LoadReinforcements(RD2) // r = 2
+        GOTO 0x27
+    LABEL 0x26
+        LoadReinforcements(RD3) // r =3
+    LABEL 0x27
+
+    RANDOMNUMBER 0x8       //Sets slot C to a number from 0 to 8? smilies says 9
+    //Now get remainder mod 4 (possibilities: 0, 1, 2, 3)
+    SMOV r1 0x4
+    SDIV slotsParam(r2, rC, r1) //r2 = rC/4
+    SMUL slotsParam(r1, r2)     //r1 = r2*4
+    SSUB slotsParam(r1, rC, r1)     //r1 -= rC, r1 = remainder
+    BNE 0x28 rC 0
```

Comment:
While these branch ladders work, it might be more legible (and thus easier to keep track of) if you used the pattern:

```
SMOV r1 0
BEQ 0 rC r1
SMOV r1 1
BEQ 1 rC r1
// . . .
LABEL 0
LoadReinforcements(blah)
GOTO 10
LABEL 1
(same)
GOTO 10

... 
LABEL 10
NoFade
ENDB
```

Considering you have SMOVs as it is, it might be worth for being able to organize the spawns separately from the checks. (also it takes linear time rather than quadratic time but we don't really care about that unless it causes noticable lag lol)